src/config.rs

```diff
@@ -94,10 +102,10 @@ mod tests {
 			jail.create_file(DEFAULT_CONFIG_FILE, indoc! { r#"
 				groups:
 				  foo:
-				    nodeSelector:
-				    	a: "1"
-				    	b: "2"
-				    	c: "3"
+                    nodeSelector:
+                      a: "1"
+                      b: "2"
+                      c: "3"
```

Comment:
For some reason all these tabs got changed to spaces. I am not sure `indoc` will handle them probably, so it's best we fix them now.